.github/workflows/auto-create-pr.yml

```
@@ -0,0 +1,47 @@
+jobs:
+  create_pr:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v4
+
+      - name: Create Draft PR to main
+        uses: peter-evans/create-pull-request@v4
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)